config/default.yml

```diff
@@ -553,6 +553,10 @@ Style/TrivialAccessors:
     - to_s
     - to_sym
 
+Style/UnneededPercentX:
+  # Enforce using %x when the command string spans multiple lines.
+  EnforcePercentXForMultiLine: false
```

Comment:
Ah, I see what you mean - if this was `AllowPercentXForMultiline` it would probably be better. Or we can rename the cop and implement distinct styles which might be the best option. As it stands the name of the cop is kind of limiting.